webots_ros2_tiago/launch/tiago.launch.py

```diff
@@ -34,19 +34,33 @@ def generate_launch_description():
     webots = launch_ros.actions.Node(package='webots_ros2_core', node_executable='webots_launcher',
                                      arguments=arguments, output='screen')
     # Controller node
-    synchronization = launch.substitutions.LaunchConfiguration('synchronization', default=False)
+    synchronization = launch.substitutions.LaunchConfiguration(
+        'synchronization', default=False)
     controller = ControllerLauncher(package='webots_ros2_tiago',
                                     node_executable='tiago_controller',
-                                    parameters=[{'synchronization': synchronization}],
+                                    parameters=[
+                                        {'synchronization': synchronization}],
                                     output='screen')
+
+    # Rviz node
+    use_rviz = launch.substitutions.LaunchConfiguration('rviz', default=False)
+    rviz_config = os.path.join(get_package_share_directory(
+        'webots_ros2_tiago'), 'resource', 'odometry.rviz')
+    rviz = launch_ros.actions.Node(package='rviz2', node_executable='rviz2', output='screen',
+                                   arguments=[
+                                       '--display-config=' + rviz_config],
+                                   condition=launch.conditions.IfCondition(use_rviz))
+
     return launch.LaunchDescription([
         webots,
         controller,
+        rviz,
```

Comment:
Rviz should be added to the dependencies: https://github.com/cyberbotics/webots_ros2/blob/0a291134bd626c49a1ded4911163ca3e8c5705f7/webots_ros2_tiago/package.xml#L17